src/torrent.cpp

```diff
@@ -10754,11 +10754,11 @@ namespace libtorrent
 		m_stats_counters.inc_stats_counter(counters::waste_piece_timed_out + static_cast<int>(reason), b);
 	}
 
-	void torrent::add_failed_bytes(int b)
+	void torrent::add_failed_bytes(int const b)
 	{
 		TORRENT_ASSERT(is_single_thread());
 		TORRENT_ASSERT(b > 0);
-		m_total_failed_bytes += b;
+		m_total_failed_bytes += std::uint32_t(b);
```

Comment:
it seems a bit questionable to use unsigned types for these members (``m_failed_bytes`` and ``m_total_redundant_bytes``). In some sense these warnings is a reminder of that